mailet/standard/src/test/java/org/apache/james/transport/mailets/AddFooterTest.java

```diff
@@ -132,28 +132,21 @@ public void shouldEnsureCarriageReturnWhenAddFooterWithTextPlainMessage() throws
                 "",
                 "Test=E0 and one =A4");
 
-        String expected = Joiner.on("\r\n").join(
-                "Subject: test",
-                "Content-Type: text/plain; charset=ISO-8859-15",
-                "MIME-Version: 1.0",
-                "Content-Transfer-Encoding: quoted-printable",
-                "",
-                "Test=E0 and one =A4",
-                "------ my footer =E0/=A4 ------");
+        String expectedFooter = "------ " + MY_FOOTER + " =E0/=A4 ------";
         
 
         Mail mail = FakeMail.fromMime(quotedPrintableTextPlainMessage, javaCharset, javaMailCharset);
         mailet.service(mail);
 
-        assertThat(MailUtil.toString(mail, javaCharset)).isEqualTo(expected);
+        assertThat(MailUtil.toString(mail, javaCharset)).endsWith(expectedFooter);
```

Comment:
the test is about carriage return, what you don't check anymore

Reply:
Sorry. It is fixed...